src/Services/MeasurementCalculationService.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class MeasurementCalculationService
+{
+    public const MM = 'mm';
+    public const CM = 'cm';
+    public const IN = 'in';
+    public const FT = 'ft';
+
+    public static function calculateDimension(int $dimension, string $unitOfMeasurement): string
```

Comment:
If you're returning a string from this function and not a number anyway, why wouldn't you include the unit of measurement as part of that string? It would simplify the code in the display method.